android/app/src/main/java/com/on/turip/ui/common/TuripUrlConverter.kt

```diff
@@ -0,0 +1,22 @@
+package com.on.turip.ui.common
+
+object TuripUrlConverter {
+    private const val YOUTUBE_VIDEO_THUMBNAIL_FORM = "https://img.youtube.com/vi/%s/0.jpg"
+    private const val VIDEO_ID_POSITION = 1
+
+    private object VideoRegex {
+        val patternVParam = "v=([a-zA-Z0-9_-]{11})(?:&|$)".toRegex()
+        val patternShortUrl = "youtu\\.be/([a-zA-Z0-9_-]{11})".toRegex()
+    }
+
+    fun extractVideoId(videoUrl: String): String {
+        val matches =
+            VideoRegex.patternVParam.find(videoUrl) ?: VideoRegex.patternShortUrl.find(videoUrl)
+        return matches?.groups?.get(VIDEO_ID_POSITION)?.value ?: ""
+    }
+
+    fun toVideoThumbnailUrl(videoUrl: String): String {
```

Comment:
c: `toXXX` 의 네이밍이라 확장함수처럼 쓰일 것 같다는 느낌 ? .,,
```suggestion
    fun String.toVideoThumbnailUrl(): String {
```
혹은 동사로 시작하는 함수의 네이밍으로 변경하는 것은 어떨까요

Reply:
반영 커밋: [98bc759](https://github.com/woowacourse-teams/2025-Turip/pull/109/commits/98bc759d29eb1cffa5905f78e651cb8f8a43f856)